$OpenBSD$

Define EM_AVR, since it isn't included in our base libelf yet.

Index: fileio.c
--- fileio.c.orig
+++ fileio.c
@@ -34,6 +34,7 @@
 #elif defined(HAVE_LIBELF_LIBELF_H)
 #include <libelf/libelf.h>
 #endif
+#define EM_AVR 83
 #define EM_AVR32 0x18ad         /* inofficial */
 #endif
 
